src/models/extra-credit/extra-credit-data-mapper-impl.ts

```diff
@@ -39,7 +41,11 @@ export class ExtraCreditDataMapperImpl extends GenericDataMapper
   ) {
     super(acl);
     super.addRBAC(
-      [this.READ, this.READ_ALL, this.CREATE, this.UPDATE],
+      [this.READ_ALL_CLASSES, this.READ_BY_UID, this.CREATE, this.READ],
```

Comment:
These new definitions need to be cast into the string properties in the auth-service.ts file.

Reply:
Are you talking about in firebase-auth.ts? Because I can't seem to find a specific file named auth-service.ts

Reply:
Yes that one